### ISSUE TITLE
OSX_SDK_VERSION or CMAKE_OSX_SYSROOT Not found when using QT

### ISSUE THREAD

**Issue by Narrorek** [ruslo/hunter#1458]
Hello

i started in clean macOS new hunter project. When added hunter_add_package(Qt) i get error 

> [hunter ** FATAL ERROR **] Can't determine SDK. Please set OSX_SDK_VERSION or CMAKE_OSX_SYSROOT in toolchain.
> [hunter ** FATAL ERROR **] [Directory:/Users/Kuba/Tmp/_Base/6974c21/78315bd/572a965/QtBuild/10]

but i have set all necessary variables becouse when i add simple 
hunter_add_package(Boost COMPONENTS regex system filesystem)

it return success.

Can someone help with this ?

```
cmake_minimum_required(VERSION 3.11)
include("cmake/HunterGate.cmake")

set(CMAKE_CXX_STANDARD 17)
set(HUNTER_STATUS_DEBUG ON)
HunterGate(
        URL "https://github.com/ruslo/hunter/archive/v0.16.15.tar.gz"
        SHA1 "6974c2150fc0d3b09de3ad1efcbf15d360647ffa"
)
```

>  Kuba@Jakubs-MacBook-Pro  ~  uname -a
Darwin Jakubs-MacBook-Pro.local 17.5.0 Darwin Kernel Version 17.5.0: Fri Apr 13 19:32:32 PDT 2018; root:xnu-4570.51.2~1/RELEASE_X86_64 x86_64
 Kuba@Jakubs-MacBook-Pro  ~  cmake -version
cmake version 3.11.2

Xcode 9.2
Thanks

**Comment by inuaimi** [comment]
I have the same issue, Have you found a solution?

**Comment by ruslo** [comment]
> Can't determine SDK. Please set OSX_SDK_VERSION or CMAKE_OSX_SYSROOT in toolchain

@inuaimi Have you tried to set it in toolchain?

**Comment by fantasticfears** [comment]
I met the same issue. What does toolchain here means? Hunter didn't find the homebrew package. The build fails given any these variables.

```
if(APPLE)
    set(OSX_SYSROOT "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk")
    set(CMAKE_OSX_SYSROOT "${OSX_SYSROOT}"
    CACHE STRING "CMAKE_OSX_SYSROOT")
    set(CMAKE_OSX_SYSROOT macosx10.14)
    set(CMAKE_OSX_DEPLOYMENT_TARGET "10.12")
endif()
```

That's very puzzling experience...

**Comment by vmsp** [comment]
I just started using Hunter and am currently having this issue when trying to use Qt.

```cmake

# CMakeLists.txt

cmake_minimum_required(VERSION 3.8 FATAL_ERROR)

include("cmake/HunterGate.cmake")
HunterGate(
  URL "https://github.com/ruslo/hunter/archive/v0.23.155.tar.gz"
  SHA1 "6421d63cc4fd78b8622e863c6b1dcabce98e1216")

project(Test)

hunter_add_package(Qt)
```

### Versions

- macOS 10.12.6
- CMake 3.14.2
- Qt 5.11.3 (downloaded by Hunter)

### Output

```
$ export HUNTER_ROOT=~/hunter
$ cmake -H. -B_builds -DHUNTER_STATUS_DEBUG=ON -DCMAKE_BUILD_TYPE=Debug

-- [hunter *** DEBUG *** 2019-04-18T22:12:20] HUNTER_ROOT detected by environment variable
-- [hunter *** DEBUG *** 2019-04-18T22:12:20] HUNTER_ROOT: /Users/vitor/hunter
-- [hunter *** DEBUG *** 2019-04-18T22:12:20] Settings (initialize):
-- [hunter *** DEBUG *** 2019-04-18T22:12:20]   HunterGate done (NO)
-- [hunter *** DEBUG *** 2019-04-18T22:12:20]   Cache init (YES)
-- [hunter *** DEBUG *** 2019-04-18T22:12:20] Single-configuration generator
-- [hunter *** DEBUG *** 2019-04-18T22:12:20] Using default cache server
-- [hunter *** DEBUG *** 2019-04-18T22:12:20] List of cache servers:
-- [hunter *** DEBUG *** 2019-04-18T22:12:20]   * https://github.com/ingenue/hunter-cache
-- [hunter *** DEBUG *** 2019-04-18T22:12:20] Settings (finalize):
-- [hunter *** DEBUG *** 2019-04-18T22:12:20]   HunterGate done (NO)
-- [hunter *** DEBUG *** 2019-04-18T22:12:20]   Cache init (YES)
-- [hunter *** DEBUG *** 2019-04-18T22:12:20] Variables from HunterGate:
-- [hunter *** DEBUG *** 2019-04-18T22:12:20]   HUNTER_GATE_ROOT: /Users/vitor/hunter
-- [hunter *** DEBUG *** 2019-04-18T22:12:20]   HUNTER_GATE_VERSION: 0.23.155
-- [hunter *** DEBUG *** 2019-04-18T22:12:20]   HUNTER_GATE_SHA1: 6421d63cc4fd78b8622e863c6b1dcabce98e1216
-- [hunter] Calculating Toolchain-SHA1
-- [hunter *** DEBUG *** 2019-04-18T22:12:20] [/Users/vitor/src/coil-desktop/_builds/_3rdParty/Hunter/toolchain] > "/usr/local/Cellar/cmake/3.14.2/bin/cmake" "-DTOOLCHAIN_INFO_FILE=/Users/vitor/src/coil-desktop/_builds/_3rdParty/Hunter/toolchain/toolchain.info.NOLF" "-DHUNTER_SELF=/Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked" "-GUnix Makefiles" "-H/Users/vitor/src/coil-desktop/_builds/_3rdParty/Hunter/toolchain" "-B/Users/vitor/src/coil-desktop/_builds/_3rdParty/Hunter/toolchain/_builds" "-DCMAKE_MAKE_PROGRAM=/usr/bin/make" "-DCMAKE_RELEASE_POSTFIX=" "-DCMAKE_DEBUG_POSTFIX=d"
-- The C compiler identification is AppleClang 9.0.0.9000039
-- The CXX compiler identification is AppleClang 9.0.0.9000039
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/vitor/src/coil-desktop/_builds/_3rdParty/Hunter/toolchain/_builds
-- [hunter *** DEBUG *** 2019-04-18T22:12:22] Already exists: /Users/vitor/hunter/_Base/6421d63/190a4a3/toolchain.info
-- [hunter] Calculating Config-SHA1
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Cache data is up-to-date
-- [hunter] HUNTER_ROOT: /Users/vitor/hunter
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] HUNTER_TOOLCHAIN_ID_PATH: /Users/vitor/hunter/_Base/6421d63/190a4a3
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] HUNTER_CONFIGURATION_TYPES: Release;Debug
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] HUNTER_BUILD_SHARED_LIBS: 
-- [hunter] [ Hunter-ID: 6421d63 | Toolchain-ID: 190a4a3 | Config-ID: 4a1c08a ]
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] load: /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/projects/Qt/hunter.cmake
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Qt versions available: [5.5.0;5.5.0-hunter-p0;5.5.0-hunter-p1;5.5.1-hunter-p0;5.5.1-hunter-p1;5.5.1-hunter-p2;5.5.1-cvpixelbuffer-p0;5.5.1-cvpixelbuffer-2-p0;5.5.1-cvpixelbuffer-2-p1;5.5.1-cvpixelbuffer-2-p2;5.5.1-cvpixelbuffer-2-p3;5.5.1-cvpixelbuffer-2-p4;5.5.1-cvpixelbuffer-2-p5;5.5.1-cvpixelbuffer-2-p6;5.5.1-cvpixelbuffer-2-p7;5.5.1-cvpixelbuffer-2-p8;5.5.1-cvpixelbuffer-2-p9;5.6.0-alpha;5.9.1-p0;5.10.1;5.11.1;5.11.3]
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Package 'Qt' CONFIGURATION_TYPES: Release;Debug
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Package 'Qt' is cacheable: YES
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Install to: /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Install
-- [hunter] QT_ROOT: /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Install (ver.: 5.11.3)
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Locking directory: /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Lock done
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Already locked: /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Locking directory: /Users/vitor/hunter/_Base/Download/Qt/5.11.3/3f6c0df
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Lock done
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Locking directory: /Users/vitor/hunter/_Base/Cache
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Lock done
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Using environment variable HOME
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Downloading DONE metafile (try #0 of 10):
-- [hunter *** DEBUG *** 2019-04-18T22:12:23]   https://raw.githubusercontent.com/ingenue/hunter-cache/master/190a4a3/Qt/__qtbase/5.11.3/3f6c0df/da39a3e/4830b34/030514d/basic-deps.DONE
-- [hunter *** DEBUG *** 2019-04-18T22:12:23]   -> /Users/vitor/hunter/_Base/Cache/meta/190a4a3/Qt/__qtbase/5.11.3/3f6c0df/da39a3e/4830b34/030514d/basic-deps.DONE
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] File not found
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Downloading DONE metafile (try #1 of 10):
-- [hunter *** DEBUG *** 2019-04-18T22:12:23]   https://raw.githubusercontent.com/ingenue/hunter-cache/master/190a4a3/Qt/__qtbase/5.11.3/3f6c0df/da39a3e/4830b34/030514d/basic-deps.DONE
-- [hunter *** DEBUG *** 2019-04-18T22:12:23]   -> /Users/vitor/hunter/_Base/Cache/meta/190a4a3/Qt/__qtbase/5.11.3/3f6c0df/da39a3e/4830b34/030514d/basic-deps.DONE
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] File not found
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Cache miss (no basic dependencies info found: /Users/vitor/hunter/_Base/Cache/meta/190a4a3/Qt/__qtbase/5.11.3/3f6c0df/da39a3e/4830b34/030514d/basic-deps.DONE)
-- [hunter *** DEBUG *** 2019-04-18T22:12:23] Keep sources for package 'Qt': NO
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] Number of physical cores: 4
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] Number of logical cores: 8
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] Number of jobs: 8
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] HUNTER_JOBS_NUMBER: 
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] HUNTER_JOBS_NUMBER (env): 
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] HUNTER_JOBS_OPTION: 8
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] Add package: Qt
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] Component: qtbase
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] Download scheme: url_sha1_qt
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] Url: https://download.qt.io/official_releases/qt/5.11/5.11.3/single/qt-everywhere-src-5.11.3.tar.xz
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] SHA1: 3f6c0df4fa7524ed11557ed55db35ef551cd2782
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] HUNTER_TLS_VERIFY: ON
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] Internal dependencies ID: 14
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] Scheme file used: /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/projects/Qt/schemes/url_sha1_qt.cmake.in
-- [hunter] Building Qt (component: qtbase)
-- [hunter *** DEBUG *** 2019-04-18T22:12:24] [/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase] > "/usr/local/Cellar/cmake/3.14.2/bin/cmake" "-C/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/cache.cmake" "-C/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/args.cmake" "-H/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase" "-B/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build" "-DCMAKE_TOOLCHAIN_FILE=/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/toolchain.cmake" "-GUnix Makefiles" "-DCMAKE_MAKE_PROGRAM=/usr/bin/make"
loading initial cache file /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/cache.cmake
loading initial cache file /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/args.cmake
-- The C compiler identification is AppleClang 9.0.0.9000039
-- The CXX compiler identification is AppleClang 9.0.0.9000039
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -- works
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -- works
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- [hunter *** DEBUG *** 2019-04-18T22:12:25] Settings (initialize):
-- [hunter *** DEBUG *** 2019-04-18T22:12:25]   HunterGate done (YES)
-- [hunter *** DEBUG *** 2019-04-18T22:12:25]   Cache init (YES)
-- [hunter *** DEBUG *** 2019-04-18T22:12:25] Scheme: url_sha1_qt
-- [hunter *** DEBUG *** 2019-04-18T22:12:25] Qt configure options (without -skip): -confirm-license;-nomake;examples;-nomake;tests;-opensource;-no-framework;-no-dbus;-platform;macx-clang;-verbose;-static;-prefix;/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Install;-no-sql-db2;-no-sql-ibase;-no-sql-mysql;-no-sql-oci;-no-sql-odbc;-no-sql-psql;-no-sql-sqlite2;-no-sql-tds;-no-fontconfig;-no-gif;-no-glib;-no-xinput2;-no-icu;-no-openssl;-qt-freetype;-qt-harfbuzz;-qt-libjpeg;-qt-libpng;-qt-pcre;-qt-xkbcommon;-qt-zlib;-debug-and-release
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build
-- [hunter *** DEBUG *** 2019-04-18T22:12:25] Configure step successful (dir: /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase)
-- [hunter *** DEBUG *** 2019-04-18T22:12:25] [/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase] > "/usr/local/Cellar/cmake/3.14.2/bin/cmake" "--build" "/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build"
/usr/local/Cellar/cmake/3.14.2/bin/cmake -S/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase -B/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build --check-build-system CMakeFiles/Makefile.cmake 0
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E cmake_progress_start /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/CMakeFiles /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/CMakeFiles/progress.marks
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f CMakeFiles/Makefile2 all
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f CMakeFiles/Qt-qtbase.dir/build.make CMakeFiles/Qt-qtbase.dir/depend
cd /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build && /usr/local/Cellar/cmake/3.14.2/bin/cmake -E cmake_depends "Unix Makefiles" /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/CMakeFiles/Qt-qtbase.dir/DependInfo.cmake --color=
Dependee "/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/CMakeFiles/Qt-qtbase.dir/DependInfo.cmake" is newer than depender "/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/CMakeFiles/Qt-qtbase.dir/depend.internal".
Dependee "/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/CMakeFiles/CMakeDirectoryInformation.cmake" is newer than depender "/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/CMakeFiles/Qt-qtbase.dir/depend.internal".
Scanning dependencies of target Qt-qtbase
/Applications/Xcode.app/Contents/Developer/usr/bin/make -f CMakeFiles/Qt-qtbase.dir/build.make CMakeFiles/Qt-qtbase.dir/build
[ 12%] Creating directories for 'Qt-qtbase'
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E make_directory /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/QtBuild/14
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E make_directory /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/QtBuild/14
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E make_directory /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Install
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E make_directory /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/Qt-qtbase-prefix/tmp
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E make_directory /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/Qt-qtbase-prefix/src/Qt-qtbase-stamp
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E make_directory /Users/vitor/hunter/_Base/Download/Qt/5.11.3/3f6c0df
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E make_directory /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/Qt-qtbase-prefix/src/Qt-qtbase-stamp
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E touch /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/Qt-qtbase-prefix/src/Qt-qtbase-stamp/Qt-qtbase-mkdir
[ 25%] Performing download step for 'Qt-qtbase'
cd /Users/vitor/hunter/_Base/Download/Qt/5.11.3/3f6c0df && /usr/local/Cellar/cmake/3.14.2/bin/cmake -E echo "skip download"
skip download
cd /Users/vitor/hunter/_Base/Download/Qt/5.11.3/3f6c0df && /usr/local/Cellar/cmake/3.14.2/bin/cmake -E touch /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/Qt-qtbase-prefix/src/Qt-qtbase-stamp/Qt-qtbase-download
[ 37%] No patch step for 'Qt-qtbase'
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E echo_append
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E touch /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/Qt-qtbase-prefix/src/Qt-qtbase-stamp/Qt-qtbase-patch
[ 50%] No update step for 'Qt-qtbase'
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E echo_append
/usr/local/Cellar/cmake/3.14.2/bin/cmake -E touch /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/Qt-qtbase-prefix/src/Qt-qtbase-stamp/Qt-qtbase-update
[ 62%] Performing configure step for 'Qt-qtbase'
cd /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/QtBuild/14 && /usr/local/Cellar/cmake/3.14.2/bin/cmake -E echo configure
configure
cd /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/QtBuild/14 && /usr/local/Cellar/cmake/3.14.2/bin/cmake -E copy /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase/Build/qt-configure.cmake /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/QtBuild/14/qt-configure.cmake
cd /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/QtBuild/14 && /usr/local/Cellar/cmake/3.14.2/bin/cmake -P /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/QtBuild/14/qt-configure.cmake

Building Qt component:
  qtbase

Dependent components:


Components to skip:
  qt3d
  qtactiveqt
  qtandroidextras
  qtcanvas3d
  qtcharts
  qtconnectivity
  qtdatavis3d
  qtdeclarative
  qtdoc
  qtdocgallery
  qtenginio
  qtfeedback
  qtgamepad
  qtgraphicaleffects
  qtimageformats
  qtlocation
  qtmacextras
  qtmultimedia
  qtnetworkauth
  qtpim
  qtpurchasing
  qtqa
  qtquickcontrols
  qtquickcontrols2
  qtremoteobjects
  qtrepotools
  qtscript
  qtscxml
  qtsensors
  qtserialbus
  qtserialport
  qtspeech
  qtsvg
  qtsystems
  qttools
  qttranslations
  qtvirtualkeyboard
  qtwayland
  qtwebchannel
  qtwebengine
  qtwebsockets
  qtwebview
  qtwinextras
  qtx11extras
  qtxmlpatterns



[hunter ** FATAL ERROR **] Can't determine SDK. Please set OSX_SDK_VERSION or CMAKE_OSX_SYSROOT in toolchain.
[hunter ** FATAL ERROR **] [Directory:/Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/QtBuild/14]

------------------------------ ERROR -----------------------------
    https://docs.hunter.sh/en/latest/reference/errors/error.incorrect.input.data.html
------------------------------------------------------------------

CMake Error at /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/modules/hunter_error_page.cmake:12 (message):
Call Stack (most recent call first):
  /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/modules/hunter_fatal_error.cmake:20 (hunter_error_page)
  /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/modules/hunter_user_error.cmake:7 (hunter_fatal_error)
  qt-configure.cmake:192 (hunter_user_error)


make[2]: *** [Qt-qtbase-prefix/src/Qt-qtbase-stamp/Qt-qtbase-configure] Error 1
make[1]: *** [CMakeFiles/Qt-qtbase.dir/all] Error 2
make: *** [all] Error 2

[hunter ** FATAL ERROR **] Build step failed (dir: /Users/vitor/hunter/_Base/6421d63/190a4a3/4a1c08a/Build/Qt/__qtbase
[hunter ** FATAL ERROR **] [Directory:/Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/projects/Qt/qtbase]

------------------------------ ERROR -----------------------------
    https://docs.hunter.sh/en/latest/reference/errors/error.external.build.failed.html
------------------------------------------------------------------

CMake Error at /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/modules/hunter_error_page.cmake:12 (message):
Call Stack (most recent call first):
  /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/modules/hunter_fatal_error.cmake:20 (hunter_error_page)
  /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/modules/hunter_download.cmake:617 (hunter_fatal_error)
  /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/projects/Qt/qtbase/hunter.cmake:142 (hunter_download)
  /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/projects/Qt/hunter.cmake:308 (include)
  /Users/vitor/hunter/_Base/Download/Hunter/0.23.155/6421d63/Unpacked/cmake/modules/hunter_add_package.cmake:62 (include)
  CMakeLists.txt:12 (hunter_add_package)


-- Configuring incomplete, errors occurred!
See also "/Users/vitor/src/coil-desktop/_builds/CMakeFiles/CMakeOutput.log".
```

**Comment by ruslo** [comment]
@vmsp From:
- https://docs.hunter.sh/en/latest/contacts.html#please-don-t

```
Please avoid adding messages to closed issues unless it’s absolutely necessary
```

> [hunter ** FATAL ERROR **] Can't determine SDK. **Please set OSX_SDK_VERSION or CMAKE_OSX_SYSROOT in toolchain.**

What is not clear in this message?
